app/assets/stylesheets/common/base/rtl.scss

```diff
@@ -1,35 +1,27 @@
 // Right to left styles.
-// *** These styles are all going to be flipped by the r2 gem ***
-// Adding the !important declaration to a rule prevents it from being flipped.
 
-// This is used to flip the .d-icon-caret-right
-@mixin rotate($degrees) {
-  transform: rotate(#{$degrees}deg);
-}
-
-// Get the right caret to point left
-.rtl .d-icon-caret-right {
-  @include rotate(180);
-}
+.rtl {
+  .d-icon-align-right,
+  .d-icon-angle-double-right,
+  .d-icon-angle-right,
+  .d-icon-arrow-right,
+  .d-icon-caret-right,
+  .d-icon-chevron-right,
+  .d-icon-hand-point-right,
+  .d-icon-quote-right {
+    transform: rotate(180deg);
+  }
 
-// Changes the triangle arrow direction in .nav-stacked .active
-// The 'left' and 'right' in these selectors is not being flipped by r2.
-.rtl .nav-stacked a.active::after {
-  border-left-color: transparent !important;
-  border-right-color: var(--secondary) !important;
-}
-.rtl code {
-  direction: ltr !important;
-  text-align: left !important;
-}
-.rtl .pull-left {
-  float: right !important;
-}
-.rtl .pull-right {
-  float: left !important;
-}
-.rtl .autocomplete {
-  left: 27px;
+  .d-icon-align-left,
+  .d-icon-angle-double-left,
+  .d-icon-angle-left,
+  .d-icon-arrow-left,
+  .d-icon-caret-left,
+  .d-icon-chevron-left,
+  .d-icon-hand-point-left,
+  .d-icon-quote-left {
+    transform: rotate(180deg);
+  }
 }
 
 .rtl .admin-customize .current-style .toggle-mobile {
```

Comment:
CSS rules for the admin page need to stay for now because we don't serve an RTL'ified version of the admin CSS bundle.